tests/test_recipe.py

```diff
@@ -122,3 +140,39 @@ def test_download(self):
                 'Downloading test_recipe from '
                 'https://www.python.org/ftp/python/3.7.4/Python-3.7.4.tgz')]
         assert m_touch.call_count == 1
+
+    def test_download_file_scheme_https(self):
+        """
+        Verifies `urlretrieve()` is being called on https downloads.
+        """
+        recipe, filename = self.get_dummy_python_recipe_for_download_tests()
+        url = recipe.url
+        with (
+                patch_urlretrieve()) as m_urlretrieve, (
+                tempfile.TemporaryDirectory()) as temp_dir:
+            recipe.ctx.setup_dirs(temp_dir)
+            assert recipe.download_file(url, filename) == filename
+        assert m_urlretrieve.call_args_list == [mock.call(
```

Comment:
Nicer to put mock.call on the next line with everything else.

Reply:
Thanks addressed :+1: